src/selectors/entities/preferences.js

```diff
@@ -138,27 +138,19 @@ export const getTheme = createShallowSelector(
         // At this point, the theme should be a plain object
 
         // If this is a system theme, find it in case the user's theme is missing any fields
-        let baseTheme = Preferences.THEMES.default;
-        if (theme.type) {
-            for (const key of Object.keys(Preferences.THEMES)) {
-                if (Preferences.THEMES[key].type === theme.type) {
-                    baseTheme = Preferences.THEMES[key];
-                    break;
-                }
-            }
+        const baseTheme = Preferences.THEMES.default;
+        if (theme.type && theme.type !== 'custom') {
+            return baseTheme;
         }
 
         for (const key of Object.keys(baseTheme)) {
             if (theme[key]) {
                 // Fix a case where upper case theme colours are rendered as black
                 theme[key] = theme[key].toLowerCase();
-            } else {
-                // This theme is missing some colours, so fall back to the default theme when necessary
-                theme[key] = baseTheme[key];
             }
         }
 
-        return theme;
+        return Object.assign({}, Preferences.THEMES.organization, theme);
```

Comment:
I think this ticket is out of date. The default theme should be "default", not "organization"

Reply:
@asaadmahmood Can you confirm please? Did you intend for missing values to be overwritten with the `default` team as it was before or with `organization` theme as indicated in the ticket? Thanks.

Reply:
Harrison is correct @mkraft. It should be the Mattermost default theme.